### ISSUE TITLE
Design: Page where user would be redirected to after accepting an invitation to a space

### ISSUE THREAD

**Issue by rohitkrai03** [openshiftio/openshift.io#3929]
When a user is invited to a space for a role he/she gets an email containing the link to accept the invitation. After accepting the invitation there should be a page where user is redirected which shows the status for invitation if it was successful or not.

Very similar to what we have for email verification - https://github.com/fabric8-ui/fabric8-ux/issues/799

**Comment by sbryzak** [comment]
Something that is currently not supported, but most likely would be useful, is if we send the resource ID (i.e. the space ID) as a parameter of the redirect request.  If this is required, please create a new issue in [fabric8-auth](https://github.com/fabric8-services/fabric8-auth) and assign to me.

**Comment by rohitkrai03** [comment]
@catrobson Any updates on this one?

**Comment by catrobson** [comment]
I think we can just bring the user into the space dashboard (openshift.io/<spacename>/analyze).  The only way they would be able to work on content that space is by being a collaborator, so bringing them into the space dashboard so they can start working makes the most sense to me rather than having them click through an additional "confirmation" page.

**Comment by catrobson** [comment]
@rohitkrai03 Fair point - for an error situation going to a page that explains the problem that follows the same pattern we have established for the e-mail validation is perfect.  I think you can just re-use that page and enter a relevant message for this situation - we would be happy to review the PR or terminology if you want to post it here.

**Comment by rohitkrai03** [comment]
@catrobson Sure. Will work on it and raise a PR. You can approve or help with the exact error message that gets displayed. Thanks.

**Comment by serenamarie125** [comment]
@rohitkrai03 has this issue been resolved by the mentioned PRs?
